bedrock/gigabit/templates/gigabit/gigabit.html

```diff
@@ -115,7 +116,7 @@ <h1 class="large">{{ _('Mozilla Gigabit Community Fund') }}</h1>
             </p>
             <p>
               {% trans %}
-              In partnership with the National Science Foundation and US Ignite, we&rsquo;re expanding the Gigabit Community Fund to three additional gigabit cities by 2018. Interested in adding your community to list of communities being considered as part of this expansion? <a href="https://docs.google.com/forms/d/1DgjPFsydrQR2H_CdNrUEqOmbrWHASFZelV7WqQsGkRg/viewform">Please fill out this interest form</a>.
+              In partnership with the National Science Foundation and US Ignite, we&rsquo;re expanding the Gigabit Community Fund to two additional gigabit cities by 2018. Interested in adding your community to list of communities being considered as part of this expansion? <a href="https://docs.google.com/forms/d/1DgjPFsydrQR2H_CdNrUEqOmbrWHASFZelV7WqQsGkRg/viewform">Please fill out this interest form</a>.
```

Comment:
This wasn't introduced in this change, but ideally the `href` here would not be hard-coded in the trans block, and should use a variable instead like elsewhere in this template.